File: app/src/main/java/com/nvvi9/giphyme/ui/listeners/FavoriteGifItemListener.java
package com.nvvi9.giphyme.ui.listeners;

import com.nvvi9.giphyme.vo.FavoriteGifItem;

public interface FavoriteGifItemListener {
    void onItemClick(FavoriteGifItem gifItem);

    boolean onItemLongClick(FavoriteGifItem gifItem);
}

File: app/src/main/java/com/nvvi9/giphyme/ui/listeners/FavoriteGifItemListener.kt
package com.nvvi9.giphyme.ui.listeners

import com.nvvi9.giphyme.vo.FavoriteGifItem

interface FavoriteGifItemListener {
    fun onItemClick(gifItem: FavoriteGifItem)
    fun onItemLongClick(gifItem: FavoriteGifItem): Boolean
}